### ISSUE TITLE
QGIS 3.4 LTR branch linux failure

### ISSUE THREAD

**Issue by ceholden** [conda-forge/qgis-feedstock#55]
Opening this to track fixing the Linux build for the 3.4 LTR branch. @SrNetoChan got the branch building on Azure by pushing a commit, but it failed on Linux. The issue was caused by a difference of openssl versions between the build and host environment.

I'm pretty sure this is due to the `qca` feedstock bringing in the wrong version because it hadn't been rerendered for `openssl=1.1`. I need to update this feedstock anyway (both for openssl 1.1 and Qt 5.6->5.9), and will try rebuilding QGIS once it's up.

**Comment by SrNetoChan** [comment]
Great. Thanks. I will check the changes you make on qca, maybe I can learn in the process. Thanks.

**Comment by lpinner** [comment]
Is this why the currently available linux build won't run? 

```
(qgis34ltr) $ qgis
qgis: error while loading shared libraries: libcrypto.so.1.1: cannot open shared object file: No such file or directory

(qgis34ltr) $ conda list|grep "^qgis\|^openssl"
openssl                   1.0.2r               h14c3975_0    conda-forge
qgis                      3.4.6            py36h065992e_0    conda-forge
```

**Comment by ocefpaf** [comment]
I believe this is solved now with #66, right? If not please re-open.
